devicetypes/smartthings/zigbee-metering-plug.src/zigbee-metering-plug.groovy

```diff
@@ -108,6 +109,17 @@ def parse(String description) {
                         map.name = "energy"
                         map.value = zigbee.convertHexToInt(it.value)/getEnergyDiv()
                         map.unit = "kWh"
+                        
+                        if (isDawonOutlet()) { 
+                            def currentEnergy = map.value
+                            def curConsum = device.currentState("powerConsumption")?.value

```

Comment:
Please rename this variable to currentPowerConsumption.

Reply:
I will rename 'curConsum' as 'currentPowerConception'.
And it will be applied to the 'Zigbee Metering Plug Power Consensus Report' DTH.

Reply:
Why did You create a new pull request ?
https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/49192

Please add You changes in this pull request (to DTH: zigbee-metering-plug.groovy)

Reply:
10days ago, you said,
"If this capability has to be exposed please create a separate DTH that will handle it (and move the fingerprints that will use it) zigbee-metering-plug-power-consumption-report.src/zigbee-metering-plug-power-consumption-report.groovy
Please direct this pull request to the "staging" branch"

So I created 'zigbee-metering-plug-power-consumption-report' DTH.
Did I understand it wrong? If so, please explain it again.

Reply:
Yes, at that time few were still unclear regarding UI metadata etc.
Ok, let's continue the review in 
https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/49192
but please: 
- make sure to remove fingerprints from this DTH  (those that You use in new DTH).
- close this pull request

Reply:
Should I close this pull request(#49017)?

Reply:
Yes please.

Reply:
OK, I close this pull request(#49017).